docs/reference/method/MongoDBGridFSBucket-downloadToStream.txt

```diff
@@ -26,15 +26,38 @@ Definition
 
    .. include:: /includes/apiargs/MongoDBGridFSBucket-method-downloadToStream-param.rst
 
-.. todo: add examples
-
 Errors/Exceptions
 -----------------
 
 .. include:: /includes/extracts/error-gridfs-filenotfoundexception.rst
 .. include:: /includes/extracts/error-invalidargumentexception.rst
 .. include:: /includes/extracts/error-driver-runtimeexception.rst
 
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   $stream = fopen('php://temp', 'w+b');
+   fwrite($stream, "foobar");
+   rewind($stream);
+
+   $id = $bucket->uploadFromStream('filename', $stream);
+
+   $destination = fopen('php://temp', 'w+b');
+
+   $bucket->downloadToStream($id, $destination);
+
+   var_dump(stream_get_contents($destination));
```

Comment:
Ack, I didn't realize `downloadToStream()` left the stream pointer at the end. You were correct to use 
`var_dump(stream_get_contents($destination, -1, 0));` here originally.